keps/sig-apps/3329-retriable-and-non-retriable-failures/README.md

```diff
@@ -200,20 +200,19 @@ job is destined to fail regardless of the number of retries. However, since
 the large workloads are often scheduled to run over night or over the
 weekend, there is no human assistance to terminate such a job early.
 
-<!--
-TODO:
-Add a paragraph for infrastructure failures, which are not always easy to determine based on exit codes.
-
-And the motivation is that some platforms allow users to run arbitrary jobs and they don't know which exit codes to filter.
--->
-
 The need for solving the problem has been emphasized by the kubernetes
 community in the issues, see: [#17244](https://github.com/kubernetes/kubernetes/issues/17244) and [#31147](https://github.com/kubernetes/kubernetes/issues/31147).
 
 Some third-party frameworks have implemented retry policies for their pods:
 - [TensorFlow Training (TFJob)](https://www.kubeflow.org/docs/components/training/tftraining/)
 - [Argo workflow](https://github.com/argoproj/argo-workflows/blob/master/examples/retry-conditional.yaml) ([example](https://github.com/argoproj/argo-workflows/blob/master/examples/retry-conditional.yaml))
 
+Additionally, some pod failures are not linked with the container runtime,
+but rather with the internal kubernetes cluster management (see:
+[Scheduling, Preemption and Eviction](https://kubernetes.io/docs/concepts/scheduling-eviction/pod-priority-preemption/)).
+Such pod failures should be recongnized as retriable and should not increment
```

Comment:
I would say something like:

Some cluster administrators just want to make these pod failures not count towards the backoffLimit